devito/ir/support/space.py

```diff
@@ -121,25 +112,24 @@ class Interval(AbstractInterval):
 
     Create an Interval of size:
 
-        dim.size + abs(upper - lower)
+        (dim.extreme_max - dim.extreme_min + 1) + (upper - lower)
```

Comment:
Just wondering why `size` is no longer used since the new codes looks like what `size` should be doing? Also was the old `abs` superfluous since we always have `upper > lower` or did something else change?

Reply:
so the ``abs`` was wrong

``dim.extreme_max - dim.extreme_min + 1`` is more correct than ``size`` (and also reflects the implementation). If we say "size", it would be like including ``upper`` and ``lower``, which instead are treated separately (they are the ``offsets`` of the Interval)